.github/config/regression-tests.yml

```diff
@@ -0,0 +1,805 @@
+repositories:
+  Elytro-eth/soul-wallet-contract:
+    forge-version: v0.3.0
+    hardhat-config: |
+      export default {
+        "paths": {
+          "sources": "contracts",
+          "tests": {
+            "solidity": "test"
+          }
+        },
+        "solidity": {
+          "version": "0.8.24",
+          "remappings": [
+            "@soulwallet-core/=lib/soulwallet-core/",
+            "@source/=contracts/",
+            "@arbitrum/nitro-contracts=lib/nitro-contracts/",
+            "@solady=lib/solady/",
+            "@solenv=lib/solenv/src/",
+            "@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/",
+            "@account-abstraction/=lib/account-abstraction/",
+            "@crypto-lib/=lib/crypto-lib/src/",
+            "forge-std/=lib/forge-std/src/",
+            "@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts/"
+          ],
+          "settings": {
+            "viaIR": true,
+            "optimizer": {
+              "enabled": true,
+              "runs": 100000
+            },
+            "evmVersion": "cancun",
+            "metadata": {
+              "bytecodeHash": "none",
+              "appendCBOR": false
+            }
+          }
+        },
+        "solidityTest": {
+          "testFail": true,
+          "rpcEndpoints": {
+            "goerli": "${GOERLI_RPC_URL}",
+            "arbitrumgoerli": "${ARBITRUM_GOERLI_RPC_URL}",
+            "optimismgoerli": "${OPTIMISM_GOERLI_RPC_URL}",
+            "etherum": "${ETHERUM_RPC_URL}",
+            "arbitrum": "${ARBITRUM_RPC_URL}",
+            "optimism": "${OPTIMISM_RPC_URL}",
+            "scrollsepolia": "${SCROLL_SEPOLIA_RPC_URL}",
+            "basesepolia": "${BASE_SEPOLIA_RPC_URL}"
```

Comment:
Are we passing these in? Ideally, we'd have fork test coverage, but I think it's also ok to ignore in first iteration.

Reply:
No, we currently don't. We can, though! I forgot to add one config entry that's already supported. We can set `repositories.*.env` to something like:
```
export GOERLI_RPC_URL=*
```
And it will be available in the env at the test command runtime. It doesn't support sourcing the values from secrets yet if we need it.

Reply:
Awesome, I think this would be good to add if it's not too much trouble